pipelines/prs.yml

```diff
@@ -60,6 +60,11 @@ resources:
   icon: docker
   source: {repository: concourse/dev}
 
+- name: unit-image
```

Comment:
I was wondering the other day about renaming this image 🤔  At least for me, the name `unit` doesn't convey much what it really brings (a common set of tools that many tasks need).

Reply:
yeah, I'd be happy to revisit the naming convention around `unit` and `dev`, but I have no bright ideas at this moment.

Reply:
yeah, I'd be happy to revisit the naming convention around `unit` and `dev`, but I have no bright ideas at this moment.

Reply:
sameer and I were confused as well. We were thinking somthing like `test-tools` or `test-utils`

Reply:
#118